src/server/middleware.ts

```diff
@@ -99,10 +101,12 @@ export function createCustomManifestMiddleware(): RequestHandler {
             useService,
             provider,
             config,
-            licenseKey
+            licenseKey,
+            shortcut
         } = {
             // Set default values
-            uuid: `demo-app-${Math.random().toString(36).substr(2, 4)}`,
+            uuid: `demo-app-${randomId}`,
+            name: `Openfin Demo App ${randomId}`,
```

Comment:
'OpenFin' rather than 'Openfin', and let's be consistent if we're call this a test app or a demo app. So either change this to "Test App", or change "openfin-test-icon.ico" to "openfin-demo-icon.ico"